RFS/docker/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Build jars on container
+#FROM amazoncorretto:11-alpine-jdk AS builder
+#COPY /gradle /rfs-build/gradle
+#COPY build.gradle /rfs-build
+#COPY gradlew /rfs-build
+#COPY settings.gradle /rfs-build
+#COPY gradle.properties /rfs-build
+#COPY /src /rfs-build/src
+#COPY /buildSrc /rfs-build/buildSrc
+#WORKDIR /rfs-build
+#RUN ./gradlew copyRuntimeJars
```

Comment:
Did you intend to leave these lines in here?

Reply:
Taking them out, were more notes for me